pkg/vmimage/data/script.sh

```diff
@@ -174,7 +204,11 @@ EOF
 # not commited with a : so that Windows users can check out the repo
 mv /etc/docker/certs.d/docker-registry.default.svc-5000 /etc/docker/certs.d/docker-registry.default.svc:5000
 
-rm /var/lib/yum/client-cert.pem /var/lib/yum/client-key.pem
+rm -fv /etc/yum.repos.d/kickstart.repo \
+  /etc/yum.repos.d/azurecore.repo \
```

Comment:
makes sense not to remove these repo files in case there's a CVE in one of the RPMs and needs fixing?

Reply:
I added the azurecore repo to the kickstart repo file, makes no sense to use 2 diff files, wdyt?